ansible/templates/safegraph_patterns-params-prod.json.j2

```diff
@@ -19,6 +19,8 @@
       "data_source": "safegraph",
       "span_length": 14,
       "end_date": "today-4",
+      "min_expected_lag": {"all": "sunday+4,4"},
+      "max_expected_lag": {"all": "sunday+4,4"},
```

Comment:
What does this mean vs (from HHS)

```
"min_expected_lag": {"all": "1"},
"max_expected_lag": {"all": "7"},
```

'cause HHS also updates weekly, so presumably we'd want to use the "sunday+" format.

Reply:
I think my checks found that HHS doesn't always update on the same day, and the documentation says "HHS issues updates to this timeseries once a week, and occasionally more often." I'll double check on whether or not to use "sunday+" for this.

Reply:
Makes sense.

If min expected lag is 1, that means that for data issued today, the most recent date available is yesterday -- is that right? But I'm getting mixed messages on what the lag should be: According to [the HHS documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/hhs.html#lag-and-backfill) lag is 0-6 days. Taking a quick look at our as-of/issue dates in the API, it looks like lag on issue is often 2 days.

Reply:
The way I've written it right now, yes. I also saw that line in HHS documentation, but from my experience lag is rarely 0 days?

Reply:
Okay, maybe the docs are out of date, then.